openvasreporting/libs/parsed_data.py

```diff
@@ -489,13 +504,13 @@ def __init__(self, vuln_id, name, threat, **kwargs):
         :raises: TypeError, ValueError
         """
         # Get info
-        cves = kwargs.get("cves", list()) or list()
-        cvss = kwargs.get("cvss", -1.0) or -1.0
-        level = kwargs.get("level", "None") or "None"
-        tags = kwargs.get("tags", dict()) or dict()
-        references = kwargs.get("references", "Uknown") or "Unknown"
-        family = kwargs.get("family", "Unknown") or "Unknown"
-        result = kwargs.get("description", "Unknown") or "Unknown"
+        cves: list[str] = kwargs.get("cves", list()) or list()
+        cvss:     float = kwargs.get("cvss", -1.0) or -1.0
+        level:      str = kwargs.get("level", "None") or "None"
+        tags:      dict = kwargs.get("tags", dict()) or dict()
+        references: str = kwargs.get("references", "Unknown") or "Unknown"
+        family:     str = kwargs.get("family", "Unknown") or "Unknown"
+        result:     str = kwargs.get("description", "Unknown") or "Unknown"
```

Comment:
Adding spaces to get `=` aligned might conflict with PEP 8
https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements